flutter_local_notifications_linux/lib/src/dbus_wrapper.dart

```diff
@@ -0,0 +1,74 @@
+import 'dart:async';
+
+import 'package:dbus/dbus.dart';
+
+/// Mockable [DBusRemoteObject] wrapper
+abstract class DBusWrapper {
+  /// Build an instance of [DBusRemoteObject]
+  void build({
+    required String destination,
+    required String path,
+  });
+
+  /// Invokes a method on this [DBusRemoteObject].
+  /// Throws [DBusMethodResponseException] if the remote side returns an error.
+  ///
+  /// If [replySignature] is provided this causes this method to throw a
+  /// [DBusReplySignatureException] if the result is successful but the returned
+  /// values do not match the provided signature.
+  Future<DBusMethodSuccessResponse> callMethod(
+    String? interface,
+    String name,
+    Iterable<DBusValue> values, {
+    DBusSignature? replySignature,
+    bool noReplyExpected = false,
+    bool noAutoStart = false,
+    bool allowInteractiveAuthorization = false,
+  });
+
+  /// Creates a stream of signal with the given [name].
+  DBusRemoteObjectSignalStream subscribeSignal(String name);
+}
+
+/// Real implementation of [DBusWrapper]
+class DBusWrapperImpl implements DBusWrapper {
+  late final DBusRemoteObject _object;
+  late final String _destination;
+
+  @override
+  void build({
+    required String destination,
+    required String path,
+  }) {
+    _destination = destination;
+    _object = DBusRemoteObject(
+      DBusClient.session(),
```

Comment:
the docs say the client returned from this (https://pub.dev/documentation/dbus/latest/dbus/DBusClient/close.html) should be closed so based on that, looks like a way to do so would need to be provided

Reply:
`flutter_local_notifications` has no way to "clean/release" resources, because there is only an initialization method. The only solution would be to make a `close()` method in the concrete `LinuxFlutterLocalNotificationsPlugin` implementation. In general, I don't think that this is critical, because the system, killing the process (for example, clicking the user on the close button), also destroys all connections, closes open files, closes sockets and does other work to release resources. I have not observed that the Dart process is still alive after the application shutdown.

Reply:
Yeah I was thinking to have it on `LinuxFlutterLocalNotificationsPlugin` but if you haven't observed the issue then perhaps we can monitor and see how it goes given it'd be a 0.1.0 release anyway